_pages/documentation/general_docs/gpu_models/gpufs.md

```diff
@@ -0,0 +1,158 @@
+
+Using the disk image and kernel created above and the square binary built above, square can be run with the following command:
+
+```
+build/VEGA_X86/gem5.opt configs/example/gpufs/mi300.py --disk-image gem5-resources/src/x86-ubuntu-gpu-ml/disk-image/x86-ubuntu-gpu-ml --kernel gem5-resources/src/x86-ubuntu-gpu-ml/vmlinux-gpu-ml --app gem5-resources/src/gpu/square/bin.default/square.default
+```
+
+In Full System the output of the simulator and the output of the simulated system are shown in two separate locations. By default, the gem5 output prints to the terminal where gem5 is run. The simulated terminal output is located in the gem5 output directory which is `m5out` by default.
+
+Once gem5 completes (or while running) the output of the Full System simulation can be seen in `m5out/system.pc.com_1.device`. For the square example, the application will print "PASSED!" to the simulated terminal output upon successful completion.
+
+#### Using Python or shell scripts
+
+Python scripts such as PyTorch, TensorFlow, etc. and shell scripts can be passed directly as the value of the `--app` command line. For example, the following minimal PyTorch application can be run directly when saved as `pytorch_test.py`:
+
+```
+#!/usr/bin/env python3
+
+import torch
+
+x = torch.rand(5, 3).to('cuda')
+y = torch.rand(3, 5).to('cuda')
+
+z = x @ y
+```
+
+For example:
+
+```
+build/VEGA_X86/gem5.opt configs/example/gpufs/mi300.py --disk-image gem5-resources/src/x86-ubuntu-gpu-ml/disk-image/x86-ubuntu-gpu-ml --kernel gem5-resources/src/x86-ubuntu-gpu-ml/vmlinux-gpu-ml --app ./pytorch_test.py
+```
+
+#### Input files
+
+The GPU model configuration files are designed to copy the file provided to the `--app` option into the simulator. **Full System gem5 cannot read files from your host system!** If your application requires input files, they must be copied into the disk image. See instructions for [extending the disk image](https://github.com/gem5/gem5-resources/blob/stable/src/x86-ubuntu-gpu-ml/BUILDING.md) for ways to do this.
+
+If your application requires input files, it is recommended to create a shell script and pass the shell script to the `--app` option. The shell script should be written with paths relative to the disk image paths as it will run within gem5. For example, if your application requires `foo.dat`, create a shell script such as:
+
+```
+#!/bin/bash
+
+# We have previously copied foo.dat to /data outside of simulation.
+cd /data
+my_gpu_app -i foo.dat
+```
+
+## Advanced Usage
+
+#### Running without KVM
+
+The AtomicSimpleCPU can also be used in situations where the host is not x86 or KVM is not available. To enable the Atomic CPU, you will need to modify your config (e.g., `configs/example/gpufs/mi300.py`) and replace `args.cpu_type = "X86KvmCPU"` with `args.cpu_type = "AtomicSimpleCPU"`.
```

Comment:
I made that the default now: https://github.com/gem5/gem5/pull/1391/ because I did not want this to read as having example commands followed by "oh by the way this probably won't work for 80% of you, add this other command line option too."

I could talk about the new flag `--kvm-perf` which turns *on* performance counters but I am not sure how useful CPU stats are for GPU simulation. So far I have not found any issues with leaving them turned off.